network/p2p/test_utils.go

```diff
@@ -147,20 +158,31 @@ func (ln *LocalNet) NewTestP2pNetwork(ctx context.Context, nodeIndex int, keys t
 	dutyStore := dutystore.New()
 	signatureVerifier := signatureverifier.NewSignatureVerifier(nodeStorage)
 
-	cfg := NewNetConfig(keys, format.OperatorID(operatorPubkey), ln.Bootnode, testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), options.Nodes)
+	cfg := NewNetConfig(keys, format.OperatorID(operatorPubkey), ln.Bootnode, ssv_testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), options.Nodes)
 	cfg.Ctx = ctx
 	cfg.Subnets = "00000000000000000000020000000000" //PAY ATTENTION for future test scenarios which use more than one eth-validator we need to make this field dynamically changing
-	cfg.NodeStorage = mock.NodeStorage{
-		MockPrivateKeyHash:              hash,
-		RegisteredOperatorPublicKeyPEMs: []string{},
+	if err != nil {
+		return nil, err
 	}
+	ns, err := storage.NewNodeStorage(logger, db)
+	if err != nil {
+		return nil, err
+	}
+
+	shares := generateShares(t, ks, ns, networkconfig.TestNetwork)
+
+	cfg.NodeStorage = ns
 	cfg.Metrics = nil
-	cfg.MessageValidator = validation.New(
-		networkconfig.TestNetwork,
-		nodeStorage.ValidatorStore(),
-		dutyStore,
-		signatureVerifier,
-	)
+	// TODO: (Alan) decide if the code in this comment is needed, else remove
+	// cfg.MessageValidator = nil //validation.New(
+	//networkconfig.TestNetwork,
+	//nodeStorage.ValidatorStore(),
+	//dutyStore,
+	//signatureVerifier,
+	//validation.WithSelfAccept(selfPeerID, true),
+	//)
```

Comment:
commenting this means we are not running msg validation in the tests.. or you added it elsewhere?

Reply:
We have it set here https://github.com/ssvlabs/ssv/blob/alan/no-fork-fix-tests/network/p2p/p2p_test.go#L60
and use here https://github.com/ssvlabs/ssv/blob/alan/no-fork-fix-tests/network/p2p/test_utils.go#L200

Reply:
I think we now can remove commented block